lib/encoder.py

```diff
@@ -836,29 +857,33 @@ def StoreEncoding(self, encoding, workdir=None):
   def ReadEncodingResult(self, encoding, scoredir=None):
     """Reads an encoding result back from storage, if present.
 
-    None is returned file does not exist."""
+    None is returned if file does not exist.
+    If scoredir is given, only that directory is searched.
```

Comment:
I haven't been too hard on this yet, but I think it would be good to start moving towards more structure in the docstrings, at least for important functions such as this one. The structure you should have is

"""One-line desc of what the method does.

Additional explanation.

Args:
  encoding: ...
  scoredir: ...

Returns:
  The encoding result or None if no file was found in the search path.
"""